package.json

```diff
@@ -39,13 +42,22 @@
     "type": "git",
     "url": "git+https://github.com/welovecoding/website.git"
   },
+  "lint-staged": {
+    "src/main/frontend/**/*.{json,jsx}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "scripts": {
     "check": "nsp check --output summary",
     "coverage:backend": "rimraf coverage && npm run dist && cross-env JASMINE_CONFIG_PATH=src/test/support/jasmine.json istanbul cover --report html ./node_modules/jasmine/bin/jasmine.js",
     "dist": "npm run init:db && npm run dist:frontend",
     "dist:frontend": "rimraf dist/frontend/js && webpack",
     "init:db": "knex migrate:latest && knex seed:run",
     "postinstall": "npm run dist",
+    "lint": "npm run lint:frontend",
+    "lint:frontend": "prettier --write \"src/main/frontend/**/*.{json,jsx}\"",
```

Comment:
Bunch of JSON files live also in the root of the project ^^

Reply:
I know but I want to introduce Prettier to the frontend of this project first. :)